tests/system/test_fim/test_synchronization/test_synchronization.py

```diff
@@ -135,7 +134,7 @@ def test_synchronization(folder_path, case, host):
     else:
         host_manager.run_command('wazuh-agent1', f'rm -rf {folder_path}')
         folder_path = f"'/{folder_path}/{folder_path}.txt'"
-        query = " select * from fim_entry where full_path='\"{}\"'".format(folder_path)
+        query = "001 select * from fim_entry where full_path='\"{}\"'".format(folder_path)
```

Comment:
We should get the agent ID instead of using a hardcoded one.

Reply:
Added in https://github.com/wazuh/wazuh-qa/pull/5089/commits/e13544c3c3b5e7fb952b25aca5aeefc7db54a659